docs/content/dagster-cloud/managing-deployments/alerts/managing-alerts-cli.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: "Managing alert policies with the dagster-cloud CLI | Dagster Docs"
+description: "Create and configure Dagster Cloud alerts using the dagster-cloud CLI."
+
+platform_type: "cloud"
+feature: "alerting"
+---
+
+# Managing alert policies with the dagster-cloud CLI
+
+<ApplicableDagsterProduct />
+
+In this guide, we'll walk you through managing Dagster Cloud alert policies using the [`dagster-cloud` CLI](/dagster-cloud/managing-deployments/dagster-cloud-cli). Alert policies apply to a full deployment (ex: `prod`), not Branch Deployments.
+
+**Don't want to work in code?** You can also define and set alert policies using the [Dagster Cloud UI](/dagster-cloud/managing-deployments/alerts/managing-alerts-in-ui).
+
+---
+
+## How it works
+
+Alert policies are defined in a YAML file and then, using the `dagster-cloud` CLI, set for a deployment. The YAML file should contain a single top-level key (`alert_policies`) and contain a list of alert policy objects. For example:
+
+```yaml file=/dagster_cloud/alerts/email_policy.yaml
+# alert_policies.yaml
+
+alert_policies:
+  - name: "email-alert-policy"
+    description: "An alert policy to email company executives during job failure."
+    tags:
+      - key: "level"
+        value: "critical"
+    event_types:
+      - "JOB_FAILURE"
+    notification_service:
+      email:
+        email_addresses:
+          - "richard.hendricks@hooli.com"
+          - "nelson.bighetti@hooli.com"
+```
+
+### Compatible event types
```

Comment:
I think having a reference page for the structure of this file could be useful. I'd also like to include the various event types in it.